src/include/daos_cont.h

```diff
@@ -42,8 +51,29 @@ extern "C" {
 /** Skips container metadata time updates on DAOS_COO_RO open, and subsequent close */
 #define DAOS_COO_RO_MDSTATS	(1U << 4)
 
+/**
+ * Before opening the container, evict current user's handles. This flag
+ * conflicts with DAOS_COO_EVICT_ALL. Using with DAOS_COO_EX is not supported
+ * at the moment.
+ *
+ * This flag is for recovery purposes and shall not be used by regular
+ * applications.
+ */
+#define DAOS_COO_EVICT		(1U << 5)
+
+/**
+ * Before opening the container, evict all handles, including other users'.
+ * This flag conflicts with DAOS_COO_EVICT. Using with DAOS_COO_RO or
+ * DAOS_COO_RW is not supported at the moment. The current user must be the
```

Comment:
im not clear on this: Using with DAOS_COO_RO or DAOS_COO_RW is not supported at the moment.
It implies if one calls daos_cont_open(DAOS_COO_EVICT_ALL) it just evicts and does not open the container?

update -- nevermind i think i understood from reading the tests. but it might be good to indicate that evict operations only are supported today for the purpose of opening the container exclusively